kNN_NBC/IO/main.cpp

```diff
@@ -17,31 +17,53 @@ int main(int argc, char** argv)
 	TCLAP::SwitchArg headersArg("r", "headers", "Indicates that csv file contains headers row that is to be ignored");
 	TCLAP::ValueArg<int> idColArg("i", "id", "Indicates that csv file contains id column", false, -1, "int");
 	TCLAP::MultiArg<int> ignoreColsArg("n", "ignore", "Specifies columns to be ignored by algorithm", false, "int");
+	TCLAP::ValueArg<int> kArg("k", "k", "K value for algorithm", true, 3, "int");
+	TCLAP::SwitchArg kPlusNNArg("+", "k-plus", "Use K+NN rather than KNN version of algorithm");
 
```

Comment:
Prośba o dodanie jeszcze flagi na strategię wyboru punktu referencyjnego (MIN_VALUE/MAX_VALUE), umknęło mi to, jak implementowałam, a nie warto na to osobnego PR zakładać.

Reply:
Może być liczbowo z informacją o dostępnych? Będzie łatwo mapować na enum

Reply:
Jasne :D